features/dd-sdk-android-trace-otel/src/main/kotlin/com/datadog/android/trace/opentelemetry/OtelTracerProvider.kt

```diff
@@ -236,6 +239,19 @@ class OtelTracerProvider internal constructor(
             return this
         }
 
+        /**
+         * Sets the trace rate limit. This is the maximum number of traces per second that will be
+         * accepted. Please not that this property is used in conjunction with the sample rate. If no sample rate
+         * is provided this property and its related logic will be ignored.
+         * @param traceRateLimit the trace rate limit as a value between 1 and Int.MAX_VALUE (default is Int.MAX_VALUE)
+         */
+        fun setTraceRateLimit(
+            @IntRange(from = 1, to = Int.MAX_VALUE.toLong()) traceRateLimit: Int
```

Comment:
why min value is 1? shouldn't it be 0 instead to be consistent with sample rate? there we allow setting 0, meaning no traces will be sent.

Reply:
that's what they have in the code base

Reply:
it is a `Math.max(1, traceLimit)` that it's being used in that class so `0` will not be taken into account.